schemas/pyproject.toml

```diff
@@ -19,6 +22,7 @@ pytest = "^7.3.1"
 twine = "^5.1.1"
```

Comment:
Out of scope for this PR, but maybe we should consider using `poetry publish` instead of `twine`?

Reply:
https://github.com/mozilla/experimenter/issues/11574